libtransmission/rpc-server.cc

```diff
@@ -772,6 +906,16 @@ static void tr_rpcSetHostWhitelistEnabled(tr_rpc_server* server, bool isEnabled)
     server->isHostWhitelistEnabled = isEnabled;
 }
 
+int tr_rpcGetRPCSocketMode(tr_rpc_server const* server)
+{
+    return server->rpc_socket_mode;
+}
```

Comment:
If `tr_sessionGetRPCSocketMode()` is removed, this can be too (here and in rpc-server.h)

Reply:
I think this is still needed in `session.cc` to save out the socket mode, right?